src/cerebellum.py

```diff
@@ -0,0 +1,114 @@
+"""Pathfinding Module"""
+import pdb
+from queue import Queue
+from typing import List, Optional
+from src.snake import Snake
+from src.board import Board
+from src.coordinate import Coordinate
+
+
+class Cerebellum:
+    """Find paths to given coordinates."""
+    def __init__(self, brain, selected_algorithm: Optional[str]) -> None:
+        self.selected_algorithm: str = selected_algorithm if selected_algorithm is not None else 'breadth_first'
+        self.brain = brain
```

Comment:
This creates a circular dependency. Pass the items you need in directly.

- [ ] Don't pass in the brain to a submodule of brain